source/sass/blocks/advantages.scss

```
@@ -0,0 +1,16 @@
+.advantages {
+  margin: 0;
+  padding: 0;
+}
+
+.advantages-slogan {
```

```suggestion
.advantages__slogan {
```

Иначе это должно быть в другом фале с названием `advantages-slogan.scss`, но это же не так, да? Это просто опечатка потому что ты где-то пишешь правильно с двумя нижними подчеркиваниями, а где-то абы как, с тире